solutions/gatekeeper-policies/naming-rules/project/README.md

```diff
@@ -6,7 +6,7 @@ This naming policy evaluates each project's name using a series of regular expre
 
 | Name | Description |
 | -------- | --------- |
-| Tenant code | Two-character code representing a tenant. Multiple tenants can be supported by using a regex that evaluates multiple codes such as this: '^(aa\|bb\|cc)' |
+| client code | Two-character code representing a client. Multiple clients can be supported by using a regex that evaluates multiple codes such as this: '^(aa\|bb\|cc)' |
 | Environment code  | One character code representing the environment for the project: `d = dev, p = production, s = sandbox, u = uat`    |
```

Comment:
@lucstjean-ssc I think we should also use this PR to accomodate for experimentation in the evironment code